provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerController.java

```diff
@@ -48,65 +53,47 @@
 
   private Practitioner.Bundle bundle(
       MultiValueMap<String, String> parameters, int page, int count) {
-    PractitionerWrapper root = search(parameters);
+    Pair<List<PractitionerWrapper>, Integer> root = search(parameters);
     LinkConfig linkConfig =
         LinkConfig.builder()
             .path("Practitioner")
             .queryParams(parameters)
             .page(page)
             .recordsPerPage(count)
-            .totalRecords(1)
+            .totalRecords(root.getRight())
             .build();
     return bundler.bundle(
         BundleContext.of(
             linkConfig,
-            Collections.singletonList(root),
+            root.getLeft(),
             transformer,
             Practitioner.Entry::new,
             Practitioner.Bundle::new));
   }
 
+  /** Builds the PractitionerWrapper and returns it. */
+  private PractitionerWrapper practitionerWrapperBuilder(
+      ProviderContactsResponse providerContactsResponse, ProviderResponse.Value providerResponse) {
+
+    return new PractitionerWrapper.PractitionerWrapperBuilder().providerResponse(ProviderResponse.builder().value(singletonList(providerResponse)).build())
```

Comment:
Reposting this comment that got wiped out by formatting:

The fact that we're creating a fake ProviderResponse object here to wrap around the single ProviderResponse.Value object is a code smell... especially since, in the transformer where this gets used later, only the first element from the list of values in the ProviderResponse is used. How about changing the wrapper to have a ProviderResponse.Value provider field instead?